CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Make debug diagnostic logger obsolete and add Trace diagnostic logger ([#1048](https://github.com/getsentry/sentry-dotnet/pull/1048))
```

Comment:
We need a sub heading (due to the tool we use for changlog in nuget):
Also, better simply the message:

```suggestion
- Introduce TraceDiagnosticLogger and obsolete DebugDiagnosticLogger ([#1048](https://github.com/getsentry/sentry-dotnet/pull/1048))
```